vignettes/item-deletion-examples.Rmd

```diff
@@ -0,0 +1,594 @@
+                            lambda_r = lambda_INT_r,
+                            lambda_f = lambda_INT_f,
+                            nu_r = nu_INT_r,
+                            nu_f = nu_INT_f,
+                            Theta_r = Theta_INT_r, 
+                            Theta_f = Theta_INT_f,
+                            pmix_ref = pmix_CESD_r,
+                            plot_contour = FALSE,
+                            n_dim = 1)
+CESD_int
+```
+ 
+Removing the second item in this 2-item subscale leads to lower $\ov{\CAI}$, and 
+the effect size of this decrease is $h^{|2} \ov{\SR}= h^{|2}\ov{\SE}=0.173$, which
+corresponds to a worsening performance with a small effect size. The adverse 
+impact ratio goes up to $1$ from $0.966$ and $h^{\rr}\CAI^{|2}=0$ for all
+indices. The effect size of the improvement in the discrepancy between the
+reference and Efocal groups on SR when item 2 is deleted is
+$\Delta^{|2} h^{\rr}\SR=0.132$, $\Delta^{|2} h^{\rr}\SE=0.062$, and 
+$\Delta^{|2} h^{\rr}\SE=0.086$.
+Given these conflicting findings and the fact that this subscale only has two
+items, it would be best to retain both items if the subscale will be used on its 
+own to make classification decisions.
+
+Now to illustrate the use of `user_specified_items`:
+
+```{r }
+CESD_pos <- item_deletion_h(cut_z = 16/60*12, 
+                            weights_item = c(rep(1, 4)), 
+                            weights_latent = 1,
+                            alpha_r = alpha_POS_r,
+                            alpha_f = alpha_POS_f,
+                            psi_r = psi_POS_r,
+                            psi_f = psi_POS_f,
+                            lambda_r = lambda_POS_r,
+                            lambda_f = lambda_POS_f,
+                            nu_r = nu_POS_r,
+                            nu_f = nu_POS_f,
+                            Theta_r = Theta_POS_r, 
+                            Theta_f = Theta_POS_f,
+                            pmix_ref = pmix_CESD_r, 
+                            plot_contour = FALSE,
+                            user_specified_items = c(1:3))
+CESD_pos
+```
+
+Specifying `plot_contour = TRUE`:
+
+```{r fig.width=7, fig.height=6}
+CESD_pos <- item_deletion_h(cut_z = 16/60*12, 
+                            weights_item = c(rep(1,4)), 
+                            weights_latent = 1,
+                            alpha_r = alpha_POS_r,
+                            alpha_f = alpha_POS_f,
+                            psi_r = psi_POS_r,
+                            psi_f = psi_POS_f,
+                            lambda_r = lambda_POS_r,
+                            lambda_f = lambda_POS_f,
+                            nu_r = nu_POS_r,
+                            nu_f = nu_POS_f,
+                            Theta_r = Theta_POS_r, 
+                            Theta_f = Theta_POS_f,
+                            pmix_ref = pmix_CESD_r, 
+                            plot_contour = TRUE)
```

Comment:
@meltemozcan right now it seems the figures are not labelled to indicate which item is deleted; is that correct?

Reply:
@marklhc yes, that is correct. I can implement that and we could incorporate it to the main branch at a later point, when merging in the AUC/ROC edits? By the way, I've pushed the function for computing ROC to the roc branch, it should be fully functional. I started modifying it to allow the user to pass a CFA fit or a PartInv object directly instead of first calling the `return_SE_SP` function first as in the current version, but it's not yet complete so I haven't pushed these recent changes yet